pom.xml

```diff
@@ -589,7 +603,11 @@
                </exclusion>
             </exclusions>
          </dependency>
-
+         <dependency>
```

Comment:
This is not required. Setting `<flyway.version>myCustomVersion</flyway.version>` is totally sufficient

Reply:
Got removed.